presto-orc/src/main/java/com/facebook/presto/orc/stream/BooleanInputStream.java

```diff
@@ -397,8 +410,20 @@ public int getUnsetBits(int batchSize)
             throws IOException
     {
         int count = 0;
-        for (int i = 0; i < batchSize; i++) {
-            count += nextBit() ? 0 : 1;
+        try {
+            count = IntStream.range(0, batchSize)
+                    .map(i -> {
+                        try {
+                            return nextBit() ? 1 : 0;
+                        }
+                        catch (IOException e) {
+                            throw new UncheckedIOException(e);
+                        }
+                    })
+                    .sum();
+        }
+        catch (UncheckedIOException e) {
+            e.printStackTrace();
```

Comment:
We cannot print the stack trace to console and drop the exception, it must be propogated.